apps/reader/src/file.ts

```diff
@@ -86,14 +86,26 @@ async function toDataUrl(url: string) {
   return readBlob((r) => r.readAsDataURL(buffer))
 }
 
+function proxyGithubUrl(url: string) {
+  if (url.includes('github.com/')) {
+    return url.replace(/^http(s?):\/\/github\.com/, '/github/download').replace('/raw/', '/')
+  }
+
+  if (url.includes('raw.githubusercontent.com/')) {
+    return url.replace('https://raw.githubusercontent.com', '/github/download')
+  }
+
+  return url
+}
+
 export async function fetchBook(url: string) {
-  const filename = /\/([^/]*\.epub)$/i.exec(url)?.[1] ?? ''
+  const filename = decodeURIComponent(/\/([^/]*\.epub)$/i.exec(url)?.[1] ?? '')
   const books = await db?.books.toArray()
   const book = books?.find((b) => b.name === filename)
 
   return (
     book ??
-    fetch(url)
+    fetch(proxyGithubUrl(url))
```

Comment:
why add this?

Reply:
Without this, download book from Github will occur CORS error.

Reply:
It's better to move this to a new PR, but idk if we actually need this.

Reply:
ok, I delete these code.